owncloudData/src/main/java/com/owncloud/android/data/sharing/shares/db/OCShareEntity.kt

```diff
@@ -115,5 +113,26 @@ data class OCShareEntity(
                 values.getAsString(ProviderTableMeta.OCSHARES_URL)
             )
         }
+
+        fun toContentValues(entity: OCShareEntity): ContentValues =
```

Comment:
Why do you create this function inside companion object, as a static method? This function is meant to transform an instance of this class so passing it as a parameter would not be needed, you could just create the function as an instance method and call it using `ocShareEntity.toContentValues()`

Reply:
You mean creating an extension `OCShareEntity.toContentValues()` yes, it could be a good alternative

Reply:
It's not about extensions, for example in Kotlin would you convert an Int to String like below?

```
val number : Int = 2

Int.toString(number)
```

No, right? You would use `number.toString()` and this is because `toString()` is an instance method of `Int` not a class one.